lib/statsd.rb

```diff
@@ -287,10 +292,20 @@ def namespace=(namespace)
   def postfix=(pf)
     case pf
     when nil, false, '' then @postfix = nil
-    else @postfix = "#{@delimiter}#{pf}"
+    else @postfix = ".#{pf}"
     end
   end
 
+  # @attribute [w] postfix
+  #   A value to be appended to the stat name after a '.'. If the value is
+  #   blank then the postfix will be reset to nil (rather than to '.').
+  def tags=(tgs)
+    case tgs
+    when nil, false, '' then @tags = nil
+    else @tags = ",#{tgs}"
+    end
+  end  
```

Comment:
`postfix` is tacking a '.' separated string onto the end of the metric name
`tags` is very similar but needs a ',' separator and over

It felt cleaner to add a new method instead of overriding the existing functionality.